bindings/core/src/response.rs

```diff
@@ -142,6 +143,10 @@ pub enum Response {
     /// - [`GetUtxoChangesByIndex`](crate::method::ClientMethod::GetUtxoChangesByIndex)
     UtxoChanges(UtxoChangesResponse),
     /// Response for:
+    /// - [`GetUtxoChangesFull`](crate::method::ClientMethod::GetUtxoChangesFull)
+    /// - [`GetUtxoChangesFullByIndex`](crate::method::ClientMethod::GetUtxoChangesFullByIndex)
+    UtxoChangesFull(UtxoChangesFullResponse),
```

Comment:
btw ... I think for the variant it's reasonable to have `full` suffixed as it is here so both variants can be next to each other.

Reply:
They are grouped logically, by parameter. See Brord's comment.

Reply:
Oh that's the response file actually, that's a bit different. Don't think it matters that much...